ContactPoliciesApi.sln

```diff
@@ -3,7 +3,23 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.31313.79
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "doppler-contact-policies-api", "doppler-contact-policies-api\doppler-contact-policies-api.csproj", "{47B0FC09-F20B-4083-B9AC-8A4F33C5DB2D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Doppler.Contact.Policies.Api", "doppler-contact-policies-api\Doppler.Contact.Policies.Api.csproj", "{47B0FC09-F20B-4083-B9AC-8A4F33C5DB2D}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "1-Presentation", "1-Presentation", "{F69FE549-15BD-4281-B22C-0169701B1837}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "2-Business-Logic", "2-Business-Logic", "{A21E0949-711F-47A5-9697-40B8692B88DE}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "3-Data-Access", "3-Data-Access", "{E2E4CE35-BD70-4346-AE09-70ED8C9DCC00}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Doppler.Contact.Policies.Business.Logic", "Doppler.Contact.Policies.Business.Logic\Doppler.Contact.Policies.Business.Logic.csproj", "{CE672774-0E02-4359-A896-34FE8FAA2F45}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Doppler.Contact.Policies.Data.Access", "Doppler.Contact.Policies.Data.Access\Doppler.Contact.Policies.Data.Access.csproj", "{2DAE9F8E-A044-4049-A434-54B3B567422B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Doppler.Contact.Policies.Api.Test", "Doppler.Contact.Policies.Api.Test\Doppler.Contact.Policies.Api.Test.csproj", "{3D9E32B6-FFE7-445E-A07B-87BF6F2E7772}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Doppler.Contact.Policies.Business.Logic.Test", "Doppler.Contact.Policies.Business.Logic.Test\Doppler.Contact.Policies.Business.Logic.Test.csproj", "{9A7F51B9-9A38-4C7B-B11C-CC61B4780B6D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Doppler.Contact.Policies.Data.Access.Test", "Doppler.Contact.Policies.Data.Access.Test\Doppler.Contact.Policies.Data.Access.Test.csproj", "{5ABE939A-29C0-48FE-BB21-B723317B63A4}"
```

Comment:
Wow, a lot of projects for a very simple API!

Reply:
I have understood it wrong! `1-Presentation`, `2-Business-Logic` and `3-Data-Access` are folders, I thought they were projects.